### ISSUE TITLE
Suicidal detector fails on external functions

### ISSUE THREAD

**Issue by wolflo** [crytic/slither#527]
If the [example](https://github.com/crytic/slither/wiki/Detector-Documentation#suicidal) function for the suicidal detector is changed from `public` to `external` the issue is no longer flagged.

```
pragma solidity ^0.5.0;
contract Suicidal{
    function kill() external{
        selfdestruct(msg.sender);
    }
}
```

`slither --version`: 0.6.12
`solc --version`: 0.5.15

**Comment by montyly** [comment]
Hi @wolflo,

Thank you for reporting this. I agree that external functions should be included in this detector, we will include them.

**Comment by montyly** [comment]
Hi @wolflo, the fix (#530) is merged in master.

It will also be available in the upcoming release.

Thank you again for reporting this issue

**Comment by wolflo** [comment]
Thanks for the fix @montyly!
